webdriver/commands/NewWindow.json

```diff
@@ -0,0 +1,67 @@
+{
+  "webdriver": {
+    "commands": {
+      "NewWindow": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/WebDriver/Commands/NewWindow",
+          "support": {
+            "chrome": {
+              "version_added": false,
+              "notes": "To enable Chromium support, include <code>{\"goog:chromeOptions\": {\"w3c\": true}}</code> in the <a href=https://developer.mozilla.org/docs/Web/WebDriver/Capabilities>capabilities</a>."
```

Comment:
@Elchi3 this isn't exactly specifc to this PR, but there's loads of these `"version_added": false` entries combined with `"To enable Chromium…"` notes across the webdriver data. I don't know what any of them mean, which makes reviewing this PR difficult. Is this a flag that our schema can't handle? Or is it genuinely unsupported and the note is superfluous?

Reply:
Is that something which could be discussed/addressed in a separate issue? This file is just in conformance with all the others in this folder.

Reply:
@whimboo that's a fair point. I'm new to reviewing WebDriver data and I was hoping there was a quick, easy answer to my question ("Yes, this makes sense and this is what it means") so I could finish reviewing your change. That obviously didn't happen.

You're right that your PR is consistent with other WebDriver data (even if that data is consistently confusing). I'm going to give this another another look over, probably merge it, and move this discussion to a separate issue or Discourse.

Thanks for your patience.

Reply:
@Elchi3 @chrisdavidmills moved my questions to https://discourse.mozilla.org/t/understanding-webdriver-compatibility-data/35521 Sorry for the noise!